pkg/controller/netflow.go

```diff
@@ -199,5 +199,5 @@ func (cont *AciController) handleNetflowPolUpdate(obj interface{}) bool {
 	labelKey := cont.aciNameForKey("nfp", key)
 	cont.apicConn.WriteApicObjects(labelKey, cont.netflowPolObjs(nfp))
 
-	return true
+	return false
```

Comment:
Can you add a comment to the top of the function indicating what the return value indicates? (true - caller should retry/requeue, false - done)

Reply:
added